hips/tiles/tile.py

```diff
@@ -95,7 +95,7 @@ def read(cls, meta: HipsTileMeta, filename: str = None) -> 'HipsTile':
 
         if meta.file_format == 'fits':
             hdu_list = fits.open(str(path))
-            data = hdu_list[0].data
+            data = hdu_list[0].data.astype('float')
```

Comment:
OK, so I'm not surprised if the drawing doesn't work out of the box with integer data.
One has to read the fineprint and experiment / think a bit whether the transform can work on integer data or whether one has to convert to float before.
@adl1995 - See http://scikit-image.org/docs/stable/user_guide/data_types.html

So my suggestion here would be:
- Move the conversion to float to later, e.g. when accessing the tile data for drawing (not on read)
- Add a test here for int and float tiles to establish the current behaviour of our code
- If  you can't figure out all int vs float related conversion issues for drawing by early tomorrow, make a reminder issue to continue working on that in a follow-up pull request next week.
(I really want this PR finished and merged by tomorrow, so that I can start playing with the code easily in master).